{"additions": 14, "auther_ref": "xrd-disable-py-slc7", "auther_sha": "b4f99c2c36d0098da539051caa3d41005397a2f5", "author": "cms-sw", "body": "CentOS7  has OpenSSL 1.0.2 and python 3.12 SSL module requires OpenSSL 1.1.1. This causes xrootd python binding to fail with error [a]. This PR proposes to disable Xrootd python bindings for `slc7`\r\n\r\n```\r\nWARNING: pip is configured with locations that require TLS/SSL, however the ssl module in Python is not available.\r\nProcessing ./bindings/python\r\n  Installing build dependencies: started\r\n  Installing build dependencies: finished with status 'error'\r\n  error: subprocess-exited-with-error\r\n  \r\n  pip subprocess to install build dependencies did not run successfully.\r\n   exit code: 1\r\n   [11 lines of output]\r\n      WARNING: pip is configured with locations that require TLS/SSL, however the ssl module in Python is not available.\r\n      WARNING: Retrying (Retry(total=4, connect=None, read=None, redirect=None, status=None)) after connection broken by 'SSLError(\"Can't connect to HTTPS URL because the SSL module is not available.\")': /simple/setuptools/\r\n      WARNING: Retrying (Retry(total=3, connect=None, read=None, redirect=None, status=None)) after connection broken by 'SSLError(\"Can't connect to HTTPS URL because the SSL module is not available.\")': /simple/setuptools/\r\n      WARNING: Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'SSLError(\"Can't connect to HTTPS URL because the SSL module is not available.\")': /simple/setuptools/\r\n      WARNING: Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'SSLError(\"Can't connect to HTTPS URL because the SSL module is not available.\")': /simple/setuptools/\r\n      WARNING: Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'SSLError(\"Can't connect to HTTPS URL because the SSL module is not available.\")': /simple/setuptools/\r\n      Could not fetch URL https://pypi.org/simple/setuptools/: There was a problem confirming the ssl certificate: HTTPSConnectionPool(host='pypi.org', port=443): Max retries exceeded with url: /simple/setuptools/ (Caused by SSLError(\"Can't connect to HTTPS URL because the SSL module is not available.\")) - skipping\r\n      ERROR: Could not find a version that satisfies the requirement setuptools>=40.8.0 (from versions: none)\r\n      ERROR: No matching distribution found for setuptools>=40.8.0\r\n      WARNING: pip is configured with locations that require TLS/SSL, however the ssl module in Python is not available.\r\n      Could not fetch URL https://pypi.org/simple/pip/: There was a problem confirming the ssl certificate: HTTPSConnectionPool(host='pypi.org', port=443): Max retries exceeded with url: /simple/pip/ (Caused by SSLError(\"Can't connect to HTTPS URL because the SSL module is not available.\")) - skipping\r\n\r\n```", "branch": "IB/CMSSW_14_2_X/py312", "changed_files": 1, "comments": 3, "commits": 1, "created_at": "1726743708", "deletions": 2, "labels": ["externals-pending", "pending-signatures", "tests-started", "orp-pending"], "number": 9416, "release-notes": [], "review_comments": 0, "state": "open", "title": "[py312][xrootd] disable python binding for slc7 due to openssl version 1.0.2", "updated_at": "1726743811", "user": "smuzaffar"}